pmt-auth/src/middleware/Passport.ts

```diff
@@ -0,0 +1,25 @@
+
+const GitHubStrategy = require("passport-github").Strategy;
+const passport = require("passport");
+
+export default class Passport {
+  init(): any {
+    return passport.use(
+      new GitHubStrategy(
+        {
+          clientID: "32c165309b81af8510b0",
+          clientSecret: "c78b52ea8eb44fa3e5e35730d726fc132667538f",
+          callbackURL: "http://localhost:5000/auth/github/callback",
```

Comment:
It might be better to store these in a separate config file or in as environment variables.

Reply:
Yeah, this one's temporary